### PR TITLE
[NEW RELEASE] Clover plugin v4.0

### PR DESCRIPTION
**Pull request by Tony-Proum**

### CODE REVIEW

**Comment by Tony-Proum** [comment]
see https://community.sonarsource.com/t/new-release-clover-plugin-4-0/13639

**Comment by Dunemaster** [comment]
@Tony-Proum  any plans to merge this?

File: clover.properties [hunk 1]
```diff
@@ -19,3 +19,9 @@ defaults.mavenArtifactId=sonar-clover-plugin
 3.1.downloadUrl=https://binaries.sonarsource.com/Distribution/sonar-clover/sonar-clover-plugin-3.1.jar
 3.1.date=2015-12-14
 3.1.changelogUrl=https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=11232&version=12871
+
+4.0.description=Makes the plugin compatible with SQ 7.3
+4.0.sqVersions=[6.7,7.3.*]
```
[the remark this answers is not in the document]
Comment (Tony-Proum) [reply clover.properties]: Thank you for the tips :smiley:

File: clover.properties [hunk 2]
```diff
@@ -19,3 +19,9 @@ defaults.mavenArtifactId=sonar-clover-plugin
...
+4.0.downloadUrl=https://dl.bintray.com/sfeir-open-source/maven/org/cheztone/sonar-clover-plugin/4.0/sonar-clover-plugin-4.0-sources.jar
+4.0.date=2019-08-27
+4.0.changelogUrl=https://github.com/sfeir-open-source/sonar-clover/releases/tag/4.0
```
[the remark this answers is not in the document]
Comment (Tony-Proum) [reply clover.properties]: I've added it

**Comment by Tony-Proum** [comment]
@Dunemaster , I'm working on it :smile:

**Pull request merged**